vm/devices/get/guest_emulation_transport/src/process_loop.rs

```diff
@@ -1814,6 +1814,7 @@ async fn request_igvm_attest(
 ) -> Result<Result<Vec<u8>, IgvmAttestError>, FatalError> {
     const ALLOCATED_SHARED_MEMORY_SIZE: usize =
         get_protocol::IGVM_ATTEST_MSG_SHARED_GPA * hvdef::HV_PAGE_SIZE_USIZE;
+    const HOST_GENERIC_ERROR_CODE: usize = 0xFFFFFFFF;
```

Comment:
this constant should be defined in the `get_protocol` crate (and renamed to something more clearly IGVM attestation related)

Reply:
Updated